src/ProjectTemplates/Web.ProjectTemplates/WebApi-CSharp.csproj.in

```diff
@@ -2,17 +2,14 @@
 
   <PropertyGroup>
     <TargetFramework>${DefaultNetCoreTargetFramework}</TargetFramework>
-    <UserSecretsId Condition="'$(IndividualAuth)' == 'True' OR '$(OrganizationalAuth)' == 'True'">aspnet-Company.WebApplication1-53bc9b9d-9d6a-45d4-8429-2a2761773502</UserSecretsId>
```

Comment:
Why remove them?
- It's interesting that developers can have user secrets
- Isn't the directory access level key used by the Visual Studio file | new experience?

Reply:
The reason I removed this is because it wasn't used in the templates [here](https://github.com/AzureAD/microsoft-identity-web/blob/master/ProjectTemplates/templates/WebApi-CSharp/Company.WebApplication1.csproj). This property used to only apply to our old implementations of IndividualB2CAuth (since IndividualLocalAuth is not an option for this template) and OrganizationalAuth. Given that we are removing the old implementation, I'm removing this property as well.

Reply:
We resolved to keep these offline